client/lib/domains/tlds/wpcom-tlds.json

```diff
@@ -0,0 +1,268 @@
+{
+    "academy": 1,
+    "accountants": 1,
```

Comment:
With the fallback in `getTld()` we'll pick up all these anyway, so we really only need to include the third level domains.

Reply:
Yeah, I thought about that and then I thought:

1. With like ~5 entries...we could just in-line the data in the function. But, that's make it very manual.
2. So, we should do a .json file like we did here.
3. And if we're doing that....   might as well do a full list?  It's 1 KB compressed and it seems like it might be useful in some other way client-side.

I'm not terribly hung up on it though. If the consensus were what you suggest, that's fine too (and saves 900 bytes compressed).

Reply:
I'd definitely vote that less is more, but I'd also be gentle with the generation script so we can add it back in if the registrability hinting would be useful later on.